### ISSUE TITLE
Disabling a user doesn't remove or hide them from groups

### ISSUE THREAD

**Issue by smsm1** [cyclestreets/cyclescape#214]
I disabled user 233, yet it didn't hide them from the group membership, which I presume needs to be done manually at the moment.

**Comment by mvl22** [comment]
User 1554 is still being shown as subscribed to thread 5216, despite them being disabled.

The user edit page has the following text next to the 'Disabled' user box:

"This will set the name to "User id (deleted)", remove their groups, thread subscriptions, profile, and offers to lead threads permanently. It will leave their messages and issues."

**Comment by nikolai-b** [comment]
I think we need to be clear and I think I've been confused before.

I introduced that text in https://github.com/cyclestreets/cyclescape/issues/815 but I made the problem worse.  If a user deletes their account then we do remove their groups etc.  When an admin disables their account it is a temporary measure which can easily be done: https://github.com/cyclestreets/cyclescape/issues/651#issuecomment-259249877

How about we change the disable text to:
> This stops a user getting emails and stops them being able to log in.  It does not remove them from any groups or thread subscriptions.

In the case of user 1554 we can delete them if you prefer.
